coverage_config_x86_64.json

```diff
@@ -1,5 +1,5 @@
 {
-    "coverage_score": 0,
+    "coverage_score": 96.8,
```

Comment:
We should exclude the `tests/integration_tests.rs` path, otherwise it counts towards the coverage.

Reply:
👍🏻  updated it